app/_src/guides/customizing-transparent-proxy-configuration-in-kubernetes-mode.md

```diff
@@ -0,0 +1,434 @@
+---
+title: Customizing Transparent Proxy Configuration in Kubernetes Mode
+---
+
+{% assign docs = "/docs/" | append: page.version %}
```

Comment:
What are these assignments? I don't think I've seen them before 🤔

Reply:
instead of doing `[link name](/docs/{{ page.version }}/...)` we can do now just `[link name]({{ docs }}/...` it's just a helper so it's easier to write links so you don't have to remember this earlier construct and can just do `{{ docs }}`